kbase-extension/static/kbase/js/api/ModelingAPI.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * Module for API calls related to metabolic modeling.  This Includes
+ * some methods for the Patric SOLR Biochem API.  This does not depend on
+ * Javascript clients.
+ *
+ * @author nconrad <nconrad@anl.gov>
+ * @public
+ *
+ * Todo:
+ * 	- Could use Bluebird promises instead
+ *  - Could make separate api methods for each service
+ */
+
+define(['jquery', 'narrativeConfig'], function($, config) {
+
+    function ModelingAPI(token) {
+        var self = this;
+
+        this.token = token;
+
+        // Used for ModelSEED/Patric Biochem
+        var SOLR_ENDPOINT = 'https://www.patricbrc.org/api/';
```

Comment:
This should probably go in src/config.json, then you can get it with config.url()